arrows/core/video_input_image_list.h

```diff
@@ -36,15 +37,18 @@
 #include <arrows/core/kwiver_algo_core_export.h>
 
 namespace kwiver {
+
 namespace arrows {
+
 namespace core {
 
-/// Video input using list of images.
-// ----------------------------------------------------------------
+// ----------------------------------------------------------------------------
 /**
- * This class implements a video input algorithm using a list of
- * images to simulate a video. Only the images are returned. This
- * algorithm produces no metadata.
+ * Video input using list of images.
```

Comment:
The separator was in the wrong place before, but we should still keep the Doxygen brief comment using `///` syntax

Reply:
I don't think we have any guidelines on "how to doxygen", but that just feels strange to me, and we aren't consistent; later *in this same file* we use:

```
/**
 * @brief Some text.
 *
 * More text.
 */
```

I suppose it could use `@brief`, though I've always found that syntax superfluous.

What is the benefit of using different comment styles for the brief and detailed documentation, and especially of making them look like different comment blocks?